recipes/pygeos/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pygeos" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9a9209bee4c902f0beb4f1c8364243d2b5a4529c97f90181ffddaf26554653ff
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - geos
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - geos
```

Comment:
@ocefpaf does `geos` need to be listed here? (eg shapely does not do it, although it is a run dependency as well?)

Reply:
geos has run_exports, so you don't have to list but it does not hurt to have it there.

Reply:
Thanks. And for geos it does not need to be wrapped in a pin_compatible?

Reply:
No need. The global pinning + run_exports should take care of that. Numpy is a special case.